src/app/static/src/app/components/genericChart/GenericChart.vue

```diff
@@ -334,6 +337,8 @@
                     result[dataSourceId] = filterData(unfilteredData, filters, selectedFilterOptions);
                 }
 
+                this.addFilteredDataWithoutPageNumber(result)
```

Comment:
This feels a bit off as it's setting a data value from inside a computed where computeds ought to be derived from data/state. Might get unexpected effects sometimes. It might be better to make `filteredDataWithoutPages` a computed (which does all the above work), and then make `chartData` a further computed which takes `filteredDataWithoutPages` as a base, and adds the page numbers.